Update My-PR-Contribution-Practice.md

ManuSquall opened the pull request:
add my full name as a link to my github profile and a bio in My-PR-Contribution-Practice .md file.

#59 

#### make a PR contribution practice

ManuSquall committed:
Update My-PR-Contribution-Practice.md

add my full name as a link to my github profile and a bioin My-PR-Contribution-Practice .md file.

chryzcode reviewed (changes requested):
Please fix in your name alphabetically.

ManuSquall committed:
Update My-PR-Contribution-Practice.md

Fix my add (name and bio) alphabetically according to the list in My-PR-Contribution-Practice.md

chryzcode reviewed (approved):
LGTM 👍

ManuSquall commented:
Thanks @chryzcodez ! 🥳🥳
